src/data/real/nnreal.lean

```diff
@@ -63,7 +63,7 @@ open_locale classical big_operators
   linear_ordered_semiring, ordered_comm_semiring, canonically_ordered_comm_semiring,
   has_sub, has_ordered_sub, has_div, inhabited]]
 def nnreal := {r : ℝ // 0 ≤ r}
-localized "notation (name := nnreal) ` ℝ≥0 ` := nnreal" in nnreal
+localized "notation (name := nnreal) `ℝ≥0` := nnreal" in nnreal
```

Comment:
What does `(name := nnreal)` mean? It is used here but not in `nnrat` notation.

Reply:
Otherwise LGTM. Thanks!
bors d+

Reply:
Localized notations require names now, see #16252. It looks like `nnrat` got merged around the same time and missed the memo.